repo_test.go

```
@@ -0,0 +1,63 @@
+package repo
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/russross/blackfriday"
+)
+
+func TestAlpha(t *testing.T) {
+	query := startQuery()
+
+	query.Find("body > ul").Each(func(_ int, s *goquery.Selection) {
+		testList(t, s)
+	})
+}
+
+func testList(t *testing.T, list *goquery.Selection) {
+	list.Find("ul").Each(func(_ int, items *goquery.Selection) {
+		testList(t, items)
+		items.RemoveFiltered("ul")
+	})
+	checkAlphabeticOrder(t, list)
+}
+
+func readme() []byte {
+	input, _ := ioutil.ReadFile("./README.md")
```

Instead of ignoring error, you could panic if it's not nil.